### PR TITLE
[Doc] Add 'UNKNOWN_EFFECT' in the realtime section

### PR DESCRIPTION
**Pull request by mbounabi**
https://jira.kisio.org/browse/NAVP-1360
[skip-ci]

### CODE REVIEW

File: documentation/slate/source/includes/realtime.md [hunk 1]
```diff
@@ -459,3 +464,112 @@ At one of the added stop area from the additional trip, the departure time of th
...
+        "base_arrival_date_time": "20190605T193200",
+        "base_departure_date_time": "20190605T193400",
+        "stop_date_time": "realtime"
+        "departure_date_time": "20190605T193400",
+        "links": [],
+    }
+    "stop_point": ⊕{11 items},
+}
+```
+In the "stop_date_time" section of the response, the parameter "stop_date_time" is "realtime" and the fields "arrival_date_time"/"departure_date_time" are equals to the fields "base_arrival_date_time"/"base_departure_date_time".
```
Comment (kadhikari) [inline documentation/slate/source/includes/realtime.md]:
```suggestion
In the "stop_date_time" section of the response, the field "stop_date_time" is "realtime" and the fields "arrival_date_time"/"departure_date_time" are equals to the fields "base_arrival_date_time"/"base_departure_date_time".
```

File: documentation/slate/source/includes/realtime.md [hunk 2]
```diff
@@ -459,3 +464,112 @@ At one of the added stop area from the additional trip, the departure time of th
...
+    "display_informations": ⊕{9 items},
+    "first_datetime": ⊕{5 items}
+    "last_datetime": ⊕{5 items},
+    "links": ⊕[4 items],
+    "route": ⊕{8 items},
+    "stop_point": ⊕{11 items},
+}
+```
+
+In the list of "date_times" available in the response, the parameter "data_freshness" is "realtime" and the field "date_time" is equal to the field "base_date_time".
```
Comment (kadhikari) [inline documentation/slate/source/includes/realtime.md]:
```suggestion
In the list of "date_times" available in the response, the field "data_freshness" is "realtime" and the field "date_time" is equal to the field "base_date_time".
```

**Review by kadhikari — APPROVED** [review]

File: documentation/slate/source/includes/realtime.md [hunk 3]
```diff
@@ -41,7 +42,11 @@ A disruption is present in the response of the endpoints described if the reques
 ```
 The effect of the disruption is `SIGNIFICANT_DELAYS`. It means that the train will arrive late at one or more stations in its journey.
 
-In the disruption, the delay can be found in the list of "impacted_stops" with the departure/arrival status set to "delayed".  
+In the disruption, the delay can be found in the list of "impacted_stops" with the departure/arrival status set to "delayed".
+
+* "base_arrival_time"/"base_departure_time" represent the scheduled arrival/departure time without taking into account the delay
+* whereas "amended_arrival_time"/"amended_departure_time" are the actual arrival/departure time, after the delay is applied
```
Comment (benoit-bst) [inline documentation/slate/source/includes/realtime.md:48]: :+1:

File: documentation/slate/source/includes/realtime.md [hunk 4]
```diff
@@ -459,3 +464,112 @@ At one of the added stop area from the additional trip, the departure time of th
...
+        "base_arrival_date_time": "20190605T193200",
+        "base_departure_date_time": "20190605T193400",
+        "stop_date_time": "realtime"
+        "departure_date_time": "20190605T193400",
+        "links": [],
+    }
+    "stop_point": ⊕{11 items},
+}
+```
+In the "stop_date_time" section of the response, the field "stop_date_time" is "realtime" and the fields "arrival_date_time"/"departure_date_time" are equals to the fields "base_arrival_date_time"/"base_departure_date_time".
```
Comment (benoit-bst) [inline documentation/slate/source/includes/realtime.md]:
```suggestion
In the "stop_date_time" section of the response, the field "stop_date_time" is "realtime" and the fields "arrival_date_time"/"departure_date_time" are equal to the fields "base_arrival_date_time"/"base_departure_date_time".
```

**Review by benoit-bst — APPROVED** [review]
Did you see the render ?
Because, for me with github option (display the rich diff), I have render problems.

**Review by GuillaumeLescure — APPROVED** [review]

**Pull request merged**